bandersnatch_vrfs/src/ring.rs

```diff
@@ -66,7 +66,9 @@ impl KZG {
     }
 
     pub fn max_keyset_size(&self) -> usize {
-        self.piop_params.keyset_part_size
+        // Hack until we fix https://github.com/w3f/ring-proof/commit/d6bd529b9d08d2c11dfe0495556932be28dbf3bf#diff-d9afcebbe3d5a30f85085cf7ce661ad8f581ae8412b4df7e3a884ff9ccbdb472R20
+        800
```

Comment:
This has the same issue as https://github.com/w3f/ring-vrf/pull/73

But now this field is public again right? Otherwise test in substrate will fail

Reply:
oops, IK probably fucked up the branch upon which I built this, thanks!

Reply:
I'd merged the wrong branch into my local branch last week.